[2025-01-07T11:55:28.926+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cashier_file_processing.etl_process_restaurant manual__2025-01-07T11:54:40.461605+00:00 [queued]>
[2025-01-07T11:55:29.040+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cashier_file_processing.etl_process_restaurant manual__2025-01-07T11:54:40.461605+00:00 [queued]>
[2025-01-07T11:55:29.044+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-01-07T11:55:29.149+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_process_restaurant> on 2025-01-07 11:54:40.461605+00:00
[2025-01-07T11:55:29.245+0000] {standard_task_runner.py:57} INFO - Started process 3806 to run task
[2025-01-07T11:55:29.304+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'cashier_file_processing', 'etl_process_restaurant', 'manual__2025-01-07T11:54:40.461605+00:00', '--job-id', '620', '--raw', '--subdir', 'DAGS_FOLDER/Cashier_file_processing.py', '--cfg-path', '/tmp/tmp10jkxcns']
[2025-01-07T11:55:29.314+0000] {standard_task_runner.py:85} INFO - Job 620: Subtask etl_process_restaurant
[2025-01-07T11:55:29.847+0000] {task_command.py:410} INFO - Running <TaskInstance: cashier_file_processing.etl_process_restaurant manual__2025-01-07T11:54:40.461605+00:00 [running]> on host 8fff7f1050bf
[2025-01-07T11:55:30.366+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cashier_file_processing' AIRFLOW_CTX_TASK_ID='etl_process_restaurant' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T11:54:40.461605+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T11:54:40.461605+00:00'
[2025-01-07T11:55:35.114+0000] {logging_mixin.py:150} INFO - No data found in the 'restaurantmodels' collection.
[2025-01-07T11:55:35.332+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-01-07T11:55:35.660+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=cashier_file_processing, task_id=etl_process_restaurant, execution_date=20250107T115440, start_date=20250107T115528, end_date=20250107T115535
[2025-01-07T11:55:35.957+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-01-07T11:55:36.412+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
